testing/jfrog-cli/APKBUILD

```diff
@@ -1,16 +1,16 @@
 # Maintainer: Gennady Feldman <gena01@gmail.com>
-pkgname=jfrog-cli-go
-_pkgname=jfrog-cli
-pkgver=1.29.0
+pkgname=jfrog-cli
+pkgver=1.29.1
 pkgrel=0
 pkgdesc="JFrog cli"
 url="https://jfrog.com/getcli/"
 arch="all"
 license="Apache-2.0"
 makedepends="go"
 options="!check"
+replaces="jfrog-cli-go"
```

Comment:
I think you have to do both replaces and provides here, right @maxice8 ?

Reply:
@maxice8 is that the right way to handle this?

Reply:
and replaces= and provides= you're renaming the package.

replaces= will make sure jfrog-cli-go is removed from the system and provides= will make sure that people that currently have jfrog-cli-go are updated to jfrog-cli

Reply:
thanks. updated